app/Config/Schema/schema.xml

```diff
@@ -3087,6 +3087,7 @@
     <field name="enable_nsf_demo" type="L" />
     <field name="disable_expiration" type="L" />
     <field name="disable_ois_sync" type="L" />
+    <field name="disable_nonadmins_grp_crt" type="L" />
```

Comment:
I realize we're limited in how long the column name can be, but I wonder if we can somehow be more descriptive. Maybe `group_create_admin_only`?